src/test/java/com/GeorgesServer/app/RequestReaderTest.java

```diff
@@ -25,10 +25,10 @@ void requestReaderReturnsAStringFromAInputStreamReader() throws IOException {
         int request = 71;
         int streamHasNotEnded = -1;
```

Comment:
Not really part of this PR, but I like this wrapping of your magic #s, I think you could do this in the RequestReader as well